src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2604,4 +2604,7 @@
   <data name="Save" xml:space="preserve">
     <value>Save</value>
   </data>
+  <data name="MultiSelect" xml:space="preserve">
+    <value>Multiselect</value>
```

Comment:
Why is there a command for this?

Reply:
This command is no longer used in the menu. However, the command selector can display commands that do not have direct access. It is one of his interests. In addition, it allows you to assign it a custom hotkey. Personally, this command will be the first to have one when possible.

Reply:
This is a setting though, not an action.

Reply:
Toggle a setting can be an action. This is very practical when you often modify certain settings.